### PR TITLE
Start adding past speakers to index.html

### PR DESCRIPTION
**Pull request by HologramOfMe**
It's a start. I will be able to do more during the week.

You will note some missing links (ex. github or twitter profile links) I've been sending messages to people to try getting hold of those.

Oh and a better image for Curtis would be good. I've linked to the header image on his personal site. Not ideal.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -243,10 +244,91 @@ <h4 class="media__body__heading">Brenton Cleeland</h4>
                                     </ul>
                                 </div>
                             </div>
+                        </article> -->
+
+		      <article class="past-speaker Grid-cell">
```
Comment (MarkusH) [inline index.html]: Can you fix the indentation here to be all spaces and not a mixture of spaces and tabs, please :)

**Comment by HologramOfMe** [comment]
Did that work? Very inexperienced with git(hub). Still trying to understand what I'm seeing. It looks to me like I have made the change and sent a renewed pull request. But I could be wrong.

File: index.html [hunk 2]
```diff
@@ -243,10 +244,91 @@ <h4 class="media__body__heading">Brenton Cleeland</h4>
                                     </ul>
                                 </div>
                             </div>
+                        </article> -->
+
+                      <article class="past-speaker Grid-cell">
+                            <div class="media">
+                                <div class="media__figure">
+                                    <img src="https://pbs.twimg.com/profile_images/1755583699/me-smaller_400x400.JPG" style="width:100px;height:100px;"/>
```
Comment (sesh) [inline index.html:253]: The `style` here should be removed and added to the `_objects.media.scss` file if required (though it should already be setting the width to 80px).

File: index.html [hunk 3]
```diff
@@ -243,10 +244,91 @@ <h4 class="media__body__heading">Brenton Cleeland</h4>
...
+                                </div>
+                                <div class="media__body">
+                                    <h4 class="media__body__heading">Anthony Menasse</h4>
+                                    <ul class="media__body__links">
+                                        <li><a href="https://twitter.com/sysantmin">@sysantmin</a></li>
+                                        <li><a href="#">github/sysantmin</a></li>
```
Comment (sesh) [inline index.html]: We need to make sure we're actually linking to the GH profile here.

File: index.html [hunk 4]
```diff
@@ -243,10 +244,91 @@ <h4 class="media__body__heading">Brenton Cleeland</h4>
...
+                                        <li><a href="#">github/colin_rolfe</a></li>
+                                    </ul>
+                                </div>
+                            </div>
+                      </article>
+
+                      <article class="past-speaker Grid-cell">
+                            <div class="media">
+                                <div class="media__figure">
+                                    <img src="http://musings.tinbrain.net/static/img/banner.png" style="width:100px;height:100px;"/>
```
Comment (sesh) [inline index.html]: A better profile image for @funkybob is his Github one:
https://avatars1.githubusercontent.com/u/301363?v=3&s=80

Notice that you can do `s=80` on the end to get a version that's 80x80 pixels.

File: index.html [hunk 5]
```diff
@@ -243,10 +244,91 @@ <h4 class="media__body__heading">Brenton Cleeland</h4>
...
+                      <article class="past-speaker Grid-cell">
+                            <div class="media">
+                                <div class="media__figure">
+                                    <img src="https://pbs.twimg.com/profile_images/344513261570777010/6a5ed3335af5d91cba7551e6d1ee6cd3_400x400.jpeg" style="width:128px;height:128px;"/>
+                                </div>
+                                <div class="media__body">
+                                    <h4 class="media__body__heading">Dan Peade</h4>
+                                    <ul class="media__body__links">
+                                        <li><a href="https://twitter.com/eedeep">@eedeep</a></li>
+                                        <li><a href="#">github/</a></li>
```
Comment (sesh) [inline index.html]: Dan's Github is: https://github.com/eedeep

**Comment by sesh** [comment]
Hey @HologramOfMe thanks for the start here! I've posted a few comments with some small fixes, but looks pretty good to me!

In the back of my mind I'm toying with the idea of powering this via some sort of API, if you're keen to work together on that we could find some time this week to have a hack!

**Pull request closed without merging**